kubernetes/helm/pinot/values.yaml

```diff
@@ -21,8 +21,8 @@
 
 image:
   repository: apachepinot/pinot
-  tag: latest-jdk11 # release-0.7.1
-  pullPolicy: IfNotPresent
+  tag: latest # release-0.7.1
```

Comment:
Does `0.7.1` still apply?

Reply:
It's just an example, let me also make it to be more update.